### PR TITLE
KITE-1080: Update to org.apache Parquet 1.8.0.

### PR DESCRIPTION
**Pull request by rdblue**

### CODE REVIEW

File: kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/MarkerRange.java [hunk 1]
```diff
@@ -201,9 +201,9 @@ private Boundary() {
     }
 
     public Boundary(MarkerComparator comparator, Marker bound, boolean isInclusive) {
-      parquet.Preconditions.checkArgument(comparator != null,
+      Preconditions.checkArgument(comparator != null,
```
Comment (tomwhite) [inline kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/MarkerRange.java]: What's the difference between the two?

**Comment by tomwhite** [comment]
One small question, but looks fine otherwise.

File: kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/MarkerRange.java (hunk 1 above)
Comment (rdblue) [inline kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/MarkerRange.java]: Parquet doesn't want to depend on guava, so it defined its own precondition helpers. It was an accident to use the Parquet one here (or anywhere outside of Parquet).

**Comment by tomwhite** [comment]
I see it now (I had it the wrong way round in my head).

+1

**Comment by rdblue** [comment]
Merged. Thanks, Tom!

**Pull request merged**